### PR TITLE
Apply new rider/resharper method body refactors

### PR DESCRIPTION
**Pull request by peppy**
These are showing up in the latest Rider beta and seem harmless enough. Make sure during review that someone checks with a non-beta release to ensure they don't show inspections there.

### CODE REVIEW

**Comment by smoogipoo** [comment]
When you say "show up", what does that mean? Are they showing up as warnings, or are they purely informational? If they're showing up as errors, I think we should turn them off, in the same way that it's turned off for normal method bodies.

**Review by smoogipoo — APPROVED** [review]

**Comment by peppy** [comment]
It's set to `warning` level on both framework and osu! projects, and doesn't look to stop compilation:

20211203 191651 (JetBrains Rider-EAP)

**Comment by smoogipoo** [comment]
That's unrelated to this PR, right?

**Comment by peppy** [comment]
Ah, wrong PR. They are showing as warnings here as well. I'm okay with turning off as well, but considering how few cases there are where we weren't already using this style it felt like leaving be was okay.

**Comment by smoogipoo** [comment]
Hmm, okay, let's see how it turns out.

**Comment by Susko3** [comment]
> Make sure during review that someone checks with a non-beta release to ensure they don't show inspections there.

Checked the changes on stable Rider 2021.2.2 and it looks good, no inspections.

File: osu.Framework.Tests/Visual/Containers/TestSceneCircularContainerSizing.cs [hunk 1]
```diff
@@ -37,11 +37,9 @@ public class TestSceneCircularContainerSizing : FrameworkTestScene
 
             AddAssert("has correct corner radius", () => hasCorrectCornerRadius);
 
-            static void onUpdate(Container parent)
-            {
+            static void onUpdate(Container parent) =>
                 // Suppose the parent has some arbitrary size prior to the child being updated...
                 parent.Size = Vector2.One;
```
Comment (frenzibyte) [inline osu.Framework.Tests/Visual/Containers/TestSceneCircularContainerSizing.cs:42]: I might be the only one here but I liked normal method bodies when the one-liner code has an inline comment attached on top of it, but it's not bothering me much so I think it's fine.

**Comment by smoogipoo** [comment]
I was a little bit partial to the other format as well, but I'm trying not to nitpick every little thing in hopes of reducing conversation on simple PRs.

**Pull request merged**